### PR TITLE
Add section for owner model. Minor linting fixes

### PR DESCRIPTION
**Pull request by dionysiosarvanitis**
Also, some minor [markdownlint](https://github.com/DavidAnson/markdownlint) fixes included.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -303,7 +317,7 @@ Please see [CONTRIBUTING](CONTRIBUTING.md) for details.
 Run the tests with:
 
 ```sh
-$ vendor/bin/phpunit
```
Comment (antonkomarev) [inline README.md]: Keep it, please.

File: README.md [hunk 2]
```diff
@@ -53,7 +53,7 @@ Laravel Ownership simplify management of eloquent model's owner. Group can be an
 First, pull in the package through Composer.
 
 ```sh
-$ composer require cybercog/laravel-ownership
```
Comment (antonkomarev) [inline README.md]: Keep it, please.

**Review by antonkomarev — CHANGES_REQUESTED** [review]
Thank you for the linting and adding new section. Just keep console related symbol `$`, please.
I don't like that on double-click it's selecting, but it shows people that command should be executed from terminal as non-sudo user.

File: README.md (hunk 2 above)
Comment (dionysiosarvanitis) [reply README.md]: No problem, reverting it. Change was based on rule [MD014](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md014).

**Review by antonkomarev — APPROVED** [review]

**Comment by antonkomarev** [comment]
@dionysiosarvanitis thank you for the contribution!

**Pull request merged**